backend/open_webui/migrations/versions/65ce764b8f7e_add_company_table.py

```
@@ -0,0 +1,36 @@
+"""Add company table
+
+Revision ID: 65ce764b8f7e
+Revises: 3781e22d8b01
+Create Date: 2025-02-10 12:34:00.054493
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+import open_webui.internal.db
```

Ruff (0.8.2)

`open_webui.internal.db` imported but unused

Remove unused import: `open_webui.internal.db`

(F401)